modules/recon/hosts-hosts/ipstack.py

```diff
@@ -22,9 +22,9 @@ def module_run(self, hosts):
             except ValueError:
                 self.error(f"Invalid JSON response for '{host}'.\n{resp.text}")
                 continue
-            region = ', '.join([str(jsonobj[x]).title() for x in ['city', 'region_name'] if jsonobj[x]]) or None
-            country = jsonobj['country_name'].title()
-            latitude = str(jsonobj['latitude'])
-            longitude = str(jsonobj['longitude'])
+            region = ', '.join([str(jsonobj[x]) for x in ['city', 'region_name'] if jsonobj[x]]) or None
```

Comment:
What about this `str`? Needed?

Reply:
Pretty sure it wouldn't matter either way since you have the or None at the end.

Reply:
Remove that one as well, and if it passes your tests, we'll go with that. There's other improvements that could be made too, but at this point lets just make sure it works.

Reply:
Looks like it passes just fine. Pushed the newest version. Agreed there are some more improvements we could be adding to a lot of the modules but getting them functioning I think is the first hurdle.